src/manifolds/HyperbolicPoincareBall.jl

```diff
@@ -0,0 +1,237 @@
+function check_manifold_point(M::Hyperbolic{N}, p::PoincareBallPoint; kwargs...) where {N}
+    mpv = check_manifold_point(Euclidean(N), p.value; kwargs...)
+    mpv === nothing || return mpv
+    if !(norm(p.value) < 1)
+        return DomainError(
+            norm(p.value),
+            "The point $(p) does not lie on $(M) since its norm is not less than 1.",
+        )
+    end
+end
+
+@doc raw"""
+    convert(::Type{PoincareBallPoint}, x::HyperboloidPoint)
```

Comment:
Why do you use `x` here instead of `p`?

Reply:
No special reason, just missed that after transferring the formula, I changed it.